apps/condo/domains/ticket/hooks/useTicketMultipleFilterLoggerConverters.ts

```diff
@@ -0,0 +1,82 @@
+import { useMemo } from 'react'
+
+import { useIntl } from '@condo/next/intl'
+
+import {
+    BuildingUnitSubType,
+    TicketSource as TicketSourceType,
+    TicketStatus as TicketStatusType,
+} from '@app/condo/schema'
+
+import { OptionType } from '@condo/domains/common/utils/tables.utils'
+import { convertToOptions } from '@condo/domains/common/utils/filters.utils'
+import { VISIBLE_TICKET_SOURCE_TYPES } from '@condo/domains/ticket/constants/common'
+import { REVIEW_VALUES } from '@condo/domains/ticket/constants'
+import { TicketSource, TicketStatus } from '@condo/domains/ticket/utils/clientSchema'
+
+
+const getOptionLabelsByValue = (options: OptionType[], keyLabel = 'label', keyValue = 'value') => {
+    return (values: string[]) => options.filter(item => values.includes(item[keyValue])).map(item => item[keyLabel])
+}
+
+export const useTicketMultipleFilterLoggerConverters = () => {
```

Comment:
I can remove the hardcoded values and send the elements as they are. But if they send ids, then when creating reports it will be difficult to find out what they mean

Reply:
removed this hook